classes/AddressFormat.php

```diff
@@ -590,6 +590,9 @@ public function getFormat($idCountry)
         if (empty($out)) {
             $out = $this->getFormatDB(Configuration::get('PS_COUNTRY_DEFAULT'));
         }
+        if (Country::isNeedDniByCountryId($idCountry) && false === strpos($out, 'dni')) {
+            $out .= "\ndni";
```

Comment:
```suggestion
            $out .= PHP_EOL . 'dni';
```

Reply:
I actually wanted to use it but since the "\n" is used in `AddressFormat::getOrderedAddressFields`
 I prefered to stick with the explicit `\n` Are we sure PHP_EOL always === "\n" on any platform?

https://github.com/PrestaShop/PrestaShop/blob/0125a82acee6942c9dc522743dee2ce0dff32e64/classes/AddressFormat.php#L518

Reply:
I'm not sure but on Windows it might be `\r\n`

Reply:
we need to change the \n everywhere we can :/ it's not cross platform

Reply:
It depends on how the field is generated and stored as well, especially on creation (the value in database seems to be `\n`)

Reply:
Ok done!